push_filter.go

```diff
@@ -16,22 +16,40 @@ import (
 	"net/url"
 )
 
-// A PushFilter represents an abstract filter for filtering out push notifications.
-type PushFilter struct {
-	Identities map[string][]EventType `msgpack:"identities" json:"identities"`
-	Params     url.Values             `msgpack:"parameters" json:"parameters"`
+// A PushFilter represents an abstract filter for filtering out push notifications. This is now aliased to PushConfig as a
+// result of re-naming the type.
+//
+// Deprecated: use the new name PushConfig instead
+type PushFilter = PushConfig
+
+// NewPushFilter returns a new PushFilter. NewPushFilter is now aliased to NewPushConfig. This was done for backwards
+// compatibility as a result of the re-naming of PushFilter to PushConfig.
+//
+// Deprecated: use the new name NewPushConfig instead
+var NewPushFilter = NewPushConfig
```

Comment:
can you make a wrapper that spits a warning message? like https://github.com/aporeto-inc/manipulate/blob/5bd82ea5b7d7643c10108e106e03d35a3726cb33/filter.go#L33

Reply:
Yeah definitely 👍 

Although I think the way with the type alias is backwards compatible?

Reply:
donezo